resources/scripts/run_prototypeSelection.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
It is also possible to include this code as part of the Snakemake smk file itself, under a `run:` declaration.